documentation/content/en/books/handbook/ports/_index.adoc

```diff
@@ -182,29 +182,29 @@ Each demonstrated command provides many switches to customize its use.
 Refer to a command's help or man page for details and more examples.
 
 [[quarterly-latest-branch]]
-=== Quarterly and Latest Ports Branches
+=== Quarterly and Latest
```

Comment:
I'm not sure removing this redundancy is wise.

Reply:
Certainly, it needs thought. Let's revisit. Critically: 

* neither `quarterly` nor `latest` is a branch. 

Amongst my overarching thoughts with this (draft) PR: 

* to some users of 14.0, Git will be a novelty, so let's aim to be **precise**; crystal clear.

Reply:
Yes and no. They *ARE* branches. There is no denying that. It's how people talk about describing the thing rather than naming it specifically.  The latest quarterly branch is a fine thing to say and well understood by most readers. It keeps us from having to update the docs every quarter and it abstracts things one level. Just like in src we have the latest stable branch.

Reply:
Thanks. Reverted with [`e39e29c` (#182)](https://github.com/freebsd/freebsd-doc/pull/182/commits/e39e29c40d6ef70cfe39b003995cd470ce163743).

(Refinements to this chapter can be made at a later date.)